docs/dashboard_ingestion_guide.md

```diff
@@ -0,0 +1,119 @@
+# Dashboard Ingestion guidance 
+(Currently this guidance is about using Databuilder to ingest Dashboard metadata into Neo4j and Elasticsearch)
+
+Dashboard ingestion consists of multiple Databuilder jobs and it can be described in four steps:
+
+ 1. Ingest base data to Neo4j.
+ 2. Ingest additional data and decorate Neo4j over base data.
+ 3. Update Elasticsearch index using Neo4j data
+ 4. Remove stale data
+
+Note that Databuilder jobs need to be sequenced as 1 -> 2 -> 3 -> 4. To sequencing these jobs, Lyft uses Airflow to orchestrate the job, but Databuilder is not limited to Airflow and you can also simply use Python script to sequence it -- not recommended for production though.
+
+Also, step 1, 3, 4 is expected to have one Databuilder job where Step 2 is expected to have **multiple** Databuilder jobs and number of Databuilder jobs in step 2 is expected to grow as we add more metadata into Dashboard. To improve performance, it is recommended, but not required, to execute Databuilder jobs in step 2 concurrently.
+
+Here this documentation will be using [Mode Dashboard](https://app.mode.com/) as concrete example to show how to ingest Dashboard metadata. However, this ingestion process not limited to Mode Dashboard and any other Dashboard can follow this flow.
+
+### 1. Ingest base data to Neo4j.
+Using [ModeDashboardExtractor](../README.md#modedashboardextractor) along with [FsNeo4jCSVLoader](../README.md#fsneo4jcsvloader) and [Neo4jCsvPublisher](../README.md#neo4jcsvpublisher) to add base information such as Dashboard group name, Dashboard group id, Dashboard group description, Dashboard name, Dashboard id, Dashboard description to Neo4j. Use [this job configuration](../README.md#modedashboardextractor) example to configure the job.
```

Comment:
maybe  a chart or a graph that illustrated how each node in the dashboard dataset model is connected?

Reply:
> starts to look at the dashboard_ingestion doc, also do you think we could an sample data with csv file for mode dashboard as a quick start? This will help to quickly bring users to understand the data relationship.

Sounds good. Let me have it in separate task.